File: examples/sizeof_termios.c

#include <stdio.h>
#include <asm/termios.h>

int main(void)
{
    printf("NCCS = %d\n", NCCS);
    printf("sizeof(termios) = %lu\n", sizeof(struct termios));
    printf("sizeof(termios2) = %lu\n", sizeof(struct termios2));
    return 0;
}

File: examples/sizeof_termios.rs
use std::mem::size_of;

fn main() {
    println!("NCCS = {}", nc::NCCS);
    println!("sizeof(termios) = {}", size_of::<nc::termios_t>());
    println!("sizeof(termios2) = {}", size_of::<nc::termios2_t>());
}
